src/__tests__/dpsl.test.js

```diff
@@ -510,4 +530,45 @@ describe('dpsl.diagnostics tests', () => {
       });
     });
   });
+
+
+  test('runSmartctlCheckRoutine works without parameters', (done) => {
+    const expectedRunRoutineResponse = {id: 123456};
+
+    const chrome = {
+      os: {
+        diagnostics: {
+          runSmartctlCheckRoutine:
+            (params = undefined) => Promise.resolve(expectedRunRoutineResponse),
+        },
+      },
+    };
+    global.chrome = chrome;
+
+    dpsl.diagnostics.nvme.runSmartctlCheckRoutine().then((routine) => {
+      expect(routine).toEqual(expectedRunRoutineResponse);
+      done();
+    });
+  });
+
+  test('runSmartctlCheckRoutine works with parameters', (done) => {
+    const expectedRunRoutineResponse = {id: 123456};
+
+    const chrome = {
+      os: {
+        diagnostics: {
+          runSmartctlCheckRoutine:
+            (params = undefined) => Promise.resolve(expectedRunRoutineResponse),
```

Comment:
Maybe here we should resolve promise only if params is expected?

Reply:
Done! :)